### DISCUSSION TITLE
[Technical Questions] Codecov check failing

### DISCUSSION THREAD

**Discussion by suleesan** [CS342/.github#50] [Technical Questions] [answered]
### In what area do you have a technical challenge?

Other (non-StanfordBDHG-related) area

### Description

For my pull request, every check passes except for Codecov, and it says I am not an activated member of this organization on Codecov. I know I activated my account and I can sign in, but the check won't re-run. I don't know how to fix this.

### Reproduction

Make a pull request.

### Expected behavior

All checks should pass.

### Code of Conduct

- [X] I agree to follow this course's Code of Conduct and Contributing Guidelines

**Comment by PSchmiedmayer** [comment, marked as answer]
@suleesan This is due to the repository being a private repository, as your project is part of the CS342 Industry track. You can discard codecov checks for now in these reports; we will fix this with a small change to the repo today. As long as all other tests are passing, you should be fine.
